src/TestRunner.js

```diff
@@ -274,7 +274,8 @@ class TestRunner {
         cache[test.testFilePath] = perf.end - perf.start;
       }
     });
-    return promisify(fs.writeFile)(cacheFile, JSON.stringify(cache));
+    return promisify(mkdirp)(path.dirname(cacheFile), {mode: '777', fs: fs})
```

Comment:
I don't think you need to pass `fs` in. We patch `graceful-fs` so `require('fs')` will always be the correct instance.